src/Microsoft.Azure.WebJobs.Logging/Internal/TimeBucket.cs

```diff
@@ -27,5 +29,63 @@ public static long ConvertToBucket(DateTime dt)
             }
             return min;
         }
+
+        // Used to compute epoch. 
+        const long ticksBaseline = 633663648000000000L;
+        const long ticksPerMonth = 26785144663987L;
+        
+        public static DateTime CommonEpoch = new DateTime(0, DateTimeKind.Utc); // maps to epoch 0 
```

Comment:
Also, this is all internal. So we could do a massive rename if somebody gets better inspiration. 

---
In reply to: [82907667](https://github.com/Azure/azure-webjobs-sdk/pull/862#discussion_r82907667) [](ancestors = 82907667,82906813)